core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -556,8 +557,19 @@ private void setupServerMetrics() {
                                 ConnectionLimitingHandler::numConnections);
         }
 
+        private void scheduleExceptionsLogging() {
+            if (!config().exceptionReportingInterval().isZero() &&
+                config().errorHandler() instanceof ExceptionReportingServerErrorHandler) {
+                ((ExceptionReportingServerErrorHandler) config.errorHandler()).start(config.workerGroup());
+            }
+        }
+
         @Override
         protected CompletionStage<Void> doStop(@Nullable Void arg) {
+            if (config().errorHandler() instanceof ExceptionReportingServerErrorHandler) {
```

Comment:
`config().errorHandler()` could be extracted into a variable.

Reply:
Ok ~ I'll extract it.